homeassistant/components/ruckus_unleashed/manifest.json

```diff
@@ -6,6 +6,6 @@
   "documentation": "https://www.home-assistant.io/integrations/ruckus_unleashed",
   "integration_type": "hub",
   "iot_class": "local_polling",
-  "loggers": ["aioruckus", "xmltodict"],
-  "requirements": ["aioruckus==0.34"]
+  "loggers": ["aioruckus", "xmltodict", "cryptography"],
```

Comment:
Not so sure about this one. Is this needed? I didn't see any logging in this library.

Reply:
I'm sorry, I'd cargo culted this setting, rather than reading the relevant HA docs.

I don't do any logging in my library. The aioruckus library uses aiohttp & I see from the aiohttp source that they use a variety of getlogger sources, but integrations never seem to list these.

Should I just remove all of the loggers entries?

Reply:
Leaving aioruckus seems fine -- generally i think we dont end up going recursive very deep, and could be ok in case debug logs are added in the future.

Reply:
I've also bumped the aioruckus version [from 0.40 to 0.41](https://github.com/ms264556/aioruckus/compare/v0.40...v0.41).
The Debian Home Assistant packaging was [blocked because of aioruckus test failures](https://github.com/ms264556/aioruckus/issues/9), so 0.41 adds mocked unit tests, rather than requiring Ruckus hardware.